README.rst

```diff
@@ -40,19 +40,15 @@ Or, to set the primary color to forest green, run::
     # Note: The nested quotes are needed in order to handle the hash (#) correctly.
     tutor config save --set 'INDIGO_PRIMARY_COLOR="#225522"'
 
-Enabling Dark Theme
+Theme Toggle Button
 -------------------
 
-To enable the dark theme for the whole platform, run::
+The theme toggle button is enabled by default when Tutor Indigo is installed. The theme can be switched from light to dark and vice versa. To disable it, run::
 
-    tutor config save --set INDIGO_ENABLE_DARK_THEME=True
+    tutor config save --set INDIGO_THEME_COOKIE_NAME=null
     tutor images build openedx
```

Comment:
The only thing left is to update the instructions :) 

    tutor config save --set INDIGO_ENABLE_DARK_TOGGLE=false
    tutor local restart lms

Reply:
Updated the instructions:
```bash
tutor config save --set INDIGO_ENABLE_DARK_TOGGLE=false
tutor images build openedx             # required after enable/disable toggle
tutor local start -d
```

Reply:
Oh it's too bad that we have to rebuild the openedx image is there any way to avoid that, such as accessing the setting from the js code?

Reply:
We are depending on rebuilding of image due to the following two reasons:
1. JS file 
2. Scss file [if condition](https://github.com/overhangio/tutor-indigo/pull/97/files#diff-d9d35a65f8c6ecdd52ae0a12630752d4aef8b9211acfe4a20346bfc16fe3f935R175) to show/hide toggle. [This can be handled in js file too.]

I'm exploring it. So far, I'm unable to find a solution. Just thinking, if we can somehow pass variable from HTML/Mako to js . something like this: but this isn't working.
```
<%!
from django.conf import settings
%>

<%
  indigoDark = settings.MFE_CONFIG.get('INDIGO_ENABLE_DARK_TOGGLE', true)
%>

<div class="theme-toggle-button mr-4">
    .....HTML.....
</div>

<script type="text/javascript">
    window.darkkTheme = '{{ indigoDark }}'
</script>
```


OR pass value to a tag and hide it like the below:
```
<%!
from django.conf import settings
%>

<%
  indigo_dark = settings.MFE_CONFIG.get('INDIGO_ENABLE_DARK_TOGGLE', true)
%>

<div class="theme-toggle-button mr-4">
    .....HTML.....
</div>
<span class='hide'>{indigo_dark}</span>
```

Reply:
Don't spend too much time on this. If you can't find a quick fix, let's just ask people to rebuild the openedx image.